scripts/ingest-tarball.sh

```diff
@@ -143,7 +143,6 @@ function cvmfs_ingest_tarball() {
 
 function check_os() {
     # Check if the operating system directory is correctly set for the contents of the tarball
-    os=$(echo "${tar_first_file}" | cut -d / -f 3)
```

Comment:
Can't make suggestions for deleted lines, apparently, but this should be something like:
```
    os=$(echo "${tar_first_file}" | cut -d / -f $(( $tar_contents_start_level + 1 )))
```